### DISCUSSION TITLE
Create More checkbox

### DISCUSSION THREAD

**Discussion by ikx94** [mnfst/llm-gateway#225] [Feature request]
QoL feature: Create More checkbox.

![image](https://github.com/user-attachments/assets/a901af35-f2ed-48c2-b355-a04bb96850c2)

This is taken from Linear. Let's say you have an entity of Colors, and you want to create 20 colors (or it could be tasks, or anything). Create More lets you stay in the creation screen after creating one, so it is much comfortable for use cases like this.

**Comment by brunobuddy** [comment]
Closing this out. It's from Manifest's old backend-as-a-service days, which we don't work on anymore since the pivot. See #742 for where things are now.
